spec/14-classes.md

```diff
@@ -1913,6 +1941,34 @@ case, the `Closure` object is empty.
 
 Closure objects can not be serialized or unserialized.
 
+###Class `DivisionByZeroError`
+
+An instance of this class is thrown when an attempt is made to divide a number by zero using the remainder operators ([`%`](10-expressions.md#multiplicative-operators) and [`%=`](10-expressions.md#compound-assignment)). The class type is defined, as follows: 
+
+```PHP
+class DivisionByZeroError extends Error
+{
+}
+```
+
+**Defined elsewhere**
+
+* [`Error`](14-classes.md#class-error)
+
+###Class `Error`
+
+This class is the base class for all internal PHP error exceptions. It is defined, as follows:
+
+```PHP
+class Error implements Throwable
+{
+}
+```
```

Comment:
@smalyshev The interface does not include properties. The protected ones are part of the API.